packages/astro/src/vite-plugin-astro-server/route.ts

```diff
@@ -240,7 +232,7 @@ export async function handleRoute({
 			renderContext = await createRenderContext({
 				request,
 				pathname,
-				env,
+				env: environment,
```

Comment:
We might as well rename `env` to `enviroment`

Reply:
I agree. I will do that when I'm refactoring RenderContext.